### PR TITLE
EPMRPP-79121 || Notifications (admin page) fixes (new field, url)

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

File: app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx [hunk 1]
```diff
@@ -159,18 +181,19 @@ export class NotificationsTab extends Component {
...
-  onEdit = (notificationCase, id) => {
+  onEdit = (notificationCase) => {
     const { showModal, tracking } = this.props;
 
     tracking.trackEvent(SETTINGS_PAGE_EVENTS.EDIT_RULE_NOTIFICATIONS);
     showModal({
       id: 'addEditNotificationCaseModal',
       data: {
-        onConfirm: (data) => this.confirmEditCase(id, data),
+        onConfirm: (data) => this.confirmEditCase({ ...data }),
```
Comment (Vadim73i) [inline app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx:196]: Maybe just data?

**Review by Vadim73i — APPROVED** [review]

**Comment by AmsterGet** [comment]
Please remove `updateProjectNotificationsConfigAction` and all related stuff as here we are migrated to the new endpoints.

File: app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx [hunk 2]
```diff
@@ -159,18 +181,19 @@ export class NotificationsTab extends Component {
...
-  onEdit = (notificationCase, id) => {
+  onEdit = (notificationCase) => {
     const { showModal, tracking } = this.props;
 
     tracking.trackEvent(SETTINGS_PAGE_EVENTS.EDIT_RULE_NOTIFICATIONS);
     showModal({
       id: 'addEditNotificationCaseModal',
       data: {
-        onConfirm: (data) => this.confirmEditCase(id, data),
+        onConfirm: (data) => this.confirmEditCase(data),
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx:196]: Just `onConfirm: this.confirmEditCase,`

**Comment by tr1ble** [comment]
> updateProjectNotificationsConfigAction

Thanks, done

> Please remove `updateProjectNotificationsConfigAction` and all related stuff as here we are migrated to the new endpoints.

Thanks, done

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**